lib/ibm/cloud_sdk.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'httparty'
+
+require_relative 'cloud/sdk_http'
+require_relative 'cloud/sdk/vpc'
+
+module IBM
+  # Holds the SDK pieces.
+  class CloudSDK
+    def initialize(api_key, logger: nil)
+      @logger = logger || Logger.new($stdout, level: :warn)
```

Comment:
@agrare What do we normally do for the default logger in client libraries? I'm pretty sure we don't use STDOUT and prefer a null logger.

Reply:
Correct we typically default to a NullLogger, STDOUT will be picked up by journald and any container log aggregation tools